js/test/root/teststrings.js

```diff
@@ -3590,7 +3590,25 @@ module.exports.teststrings = {
         test.equal(str.formatChoice(15), "Default items");
         test.done();
     },
-    
+    testStringFormatChoiceCharClassesComplex_tg_TJ: function(test) {
+        test.expect(2);
+        var str = new IString("0#There are no items.|one#There items are one|few#The items are few|many#The items are many|#Default items");
+        str.setLocale("tg-TJ");
+        test.ok(str !== null);
+
+        test.equal(str.formatChoice(1), "There items are one");
+        test.done();
+    },
+    testStringFormatChoiceCharClassesComplex_tg_TJ2: function(test) {
+        test.expect(2);
+        var str = new IString("0#There are no items.|one#There items are one|few#The items are few|many#The items are many|#Default items");
+        str.setLocale("tg-TJ");
+        test.ok(str !== null);
+
+        test.equal(str.formatChoice(7), "Default items");
```

Comment:
There are no plurals in Tajik, so both should say "Default items"

Reply:
I think the test's Input parameter string for IString shouldn't have one category. I updated the test cases.